deploy/rbd/kubernetes/v1.14+/csi-rbdplugin-provisioner.yaml

```diff
@@ -180,6 +182,9 @@ spec:
         - name: ceph-csi-config
           configMap:
             name: ceph-csi-config
+        - name: ceph-csi-encryption-kms-config
```

Comment:
as this is configmap mounting to the plugin, I would like to discuss below the point


This is not directly related to this PR

* Do we need to support multiple CO  in ceph-csi (other than CO built on kubernetes)?
if not remove the configmap attaching as a volume in deployment and use kubernetes API to fetch the configmap
* Even updating  of configmap needed driver restart I believe
CC @humblec @ShyamsundarR @nixpanic @travisn

Reply:
> as this is configmap mounting to the plugin, I would like to discuss below the point
> 
> This is not directly related to this PR
> 
>     * Do we need to support multiple CO  in ceph-csi (other than CO built on kubernetes)?

We should keep the implementation as close to the ideal as possible, so yes to the above.

>       if not remove the configmap attaching as a volume in deployment and use kubernetes API to fetch the configmap
> 
>     * Even updating  of configmap needed driver restart I believe

No, [updates](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically) to the config map would be reflected within the pod in time.

>       CC @humblec @ShyamsundarR @nixpanic @travisn